mobile/src/main/java/org/openhab/habdroid/ui/widget/WidgetImageView.java

```diff
@@ -87,24 +87,23 @@ public WidgetImageView(Context context, AttributeSet attrs, int defStyle) {
             a.recycle();
         }
 
-        mDefaultSvgSize =
-                context.getResources().getDimensionPixelSize(R.dimen.svg_image_default_size);
+        mDefaultSvgSize = context.getResources().getDimensionPixelSize(R.dimen.svg_image_default_size);
         mRefreshHandler = new RefreshHandler(this);
     }
 
-    public void setImageUrl(Connection connection, String url) {
-        setImageUrl(connection, url, false);
+    public void setImageUrl(Integer size, Connection connection, String url) {
+        setImageUrl(size, connection, url, false);
     }
```

Comment:
I'd prefer putting size as last parameter, to keep parameters ordered by importance.

Reply:
Will do